boards/radxa/0001-rockpi4-rk3399-Configure-SPI-flash-boot-offset.patch

```diff
@@ -0,0 +1,135 @@
+diff --git a/arch/arm/dts/rk3399-rock-pi-4-u-boot.dtsi b/arch/arm/dts/rk3399-rock-pi-4-u-boot.dtsi
```

Comment:
Rename to something like "Add support for SPI", as it's not strictly about the offset.